src/OrleansTestingHost/DefaultSiloBuilderFactory.cs

```diff
@@ -15,7 +18,46 @@ public ISiloHostBuilder CreateSiloBuilder(string siloName, ClusterConfiguration
             return builder.ConfigureSiloName(siloName)
                 .UseConfiguration(clusterConfiguration)
                 .ConfigureLogging(loggingBuilder => TestingUtils.ConfigureDefaultLoggingBuilder(loggingBuilder,
-                    clusterConfiguration.GetOrCreateNodeConfigurationForSilo(siloName).TraceFileName));
+                    GetLegacyTraceFileName(siloName, DateTime.UtcNow)));
+        }
+
+        internal static string GetLegacyTraceFileName(string nodeName, DateTime timestamp, string traceFileFolder = "logs", string traceFilePattern = "{0}-{1}.log")
+        {
```

Comment:
Hmmm, this is an odd name for a non-legacy feature. Also it's in an odd class.
Also, there is a ton of logic here that doesn't make sense. Probably the easiest trace file name is to concatenate `$"logs\\{DeploymentId}_{nodeName}"`
The deployment ID should already have the timestamp by default

Reply:
Why is this not a legacy feature? I marked it as internal since I think we are moving away from supporting trace file name configuration, which is part of ITraceConfiguration. User should configure their own file name from now on. 

and it is an internal method. Actually it maybe should be a private method.

Reply:
What I mean, having a log file per node & client is not legacy, we want that moving forward, and not just as an internal way of doing things. Our end users would need this working easily by default. It should be an accessible utility. Could even be a public static utility in `TestClusterOptions` if there's no better place to put it

Reply:
OK it make sense. Will make it an public utility method. Part of the purpose of this PR is also testing water on what we should continue supporting as legacy or what not. Since I'm definitely in the other end of more breaking change and less supporting legacy

Reply:
BTW I reread your comment. Yeah there's a lot of logic here which doesn't make sense but just for context that it is copied from old setTraceFileName function. 
But I think $"logs\\{DeploymentId}_{nodeName}" is good enough and much clearer here.